src/server/model/index.ts

```diff
@@ -3,7 +3,8 @@ import { z } from "zod";
 
 export const seniorSchema = z.object({
```

Comment:
Can we potentially use this instead of redefining `postSeniorSchema` and `patchSeniorSchema`?